### ISSUE TITLE
fix inlineSVG bugs

### ISSUE THREAD

**Issue by timelyportfolio** [hadley/svglite#66]
I know that `svglite` is out of development cycle, but there are two bugs with the `inlineSVG` function.

- `height`\\`width` conflict 

```
xmlSVG({plot(1:10)},width=10)
```

```
Error in inlineSVG(code, ..., standalone = standalone, height = 7, width = 7) : 
  formal argument "width" matched by multiple actual arguments
```

- `editSVG` does not create file, so cannot open the string (see #56)

Should I submit a pull to fix, or wait until the next cycle?

**Comment by timelyportfolio** [comment]
- [ ]  `editSVG` fixed now with 8bc198085339089bc3e546d26daade318f3b29f3

**Comment by timelyportfolio** [comment]
thanks!!!
